src/app/config/config.ts

```diff
@@ -33,6 +34,12 @@ const compulsoryVars = convict(compulsoryVarsSchema)
   .validate({ allowed: 'strict' })
   .getProperties()
 
+// Load and validate sgid configuration values
+// If environment variables are not present, an error will be thrown
+const sgid = convict(sgidVarsSchema)
```

Comment:
nit: prefer if this instantiation resides in `app/config/features`, following the other features